local_planner/src/nodes/star_planner.cpp

```diff
@@ -1,23 +1,23 @@
 #include "star_planner.h"
 
-StarPlanner::StarPlanner() {}
+namespace avoidance {
+
+StarPlanner::StarPlanner() : tree_age_(0) {}
 
 StarPlanner::~StarPlanner() {}
 
 // set parameters changed by dynamic rconfigure
 void StarPlanner::dynamicReconfigureSetStarParams(
-    avoidance::LocalPlannerNodeConfig& config, uint32_t level) {
+    const avoidance::LocalPlannerNodeConfig& config, uint32_t level) {
```

Comment:
Do we have a convention for the spacing? You seem to have changed the "&" to prepend the name instead of directly following the type, but it's not consistent across the code base. I don't care which convention we use, but I would prefer consistency

Reply:
we have a [script](https://github.com/PX4/avoidance/blob/master/tools/fix_style.sh) to style with clang-format. To run it: `$ ./tools/fix_style.sh local_planner`.  We would even have automatic build and style checks with [travis](https://github.com/PX4/avoidance/blob/master/.travis.yml#L72) but it isn't active

Reply:
Yes, this isn't my 'ideal' formatting style in terms of readability, but at least it is consistent and automated.